common/datetime.c

```diff
@@ -260,3 +260,57 @@ void oe_datetime_log(const char* msg, const oe_datetime_t* date)
         OE_TRACE_VERBOSE("%s %s\n", msg, str);
     }
 }
+
+oe_result_t oe_datetime_to_time_t(const oe_datetime_t* datetime, time_t* value)
+{
+    oe_result_t result = OE_FAILURE;
+    time_t now;
+    time_t tmp;
+    struct tm timeinfo;
+
+    if (datetime == NULL || value == NULL)
+        OE_RAISE(OE_INVALID_PARAMETER);
+
+    OE_CHECK(oe_datetime_is_valid(datetime));
+
+    // Get current time first
+    time(&now);
```

Comment:
Why getting current time first? Most of the fields are replaced between line 281 and 286. The fields that are not replaced do not necessarily have the same value as current.

Reply:
Yes, getting current time is not essential. 
I will update.